DMTR-102.tex

```diff
@@ -1,40 +1,36 @@
 \documentclass[DM,lsstdraft,STR,toc]{lsstdoc}
 \usepackage{geometry}
 \usepackage{longtable,booktabs}
+\usepackage{enumitem}
+\usepackage{arydshln}
 
 \input meta.tex
 
+\providecommand{\tightlist}{
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+
 \begin{document}
 
-\def\milestoneName{Raw Image Archiving}
-\def\milestoneId{LDM-503-8b}
-\def\product{LSST Raw Image Archiving System}
+\def\milestoneName{Raw image archiving for data from ccob device}
+\def\milestoneId{503-8b}
+\def\product{Archiving}
 
 \setDocCompact{true}
 
 \title[\milestoneId{}~Test Report]{\milestoneId{} (\milestoneName{})~Test Plan and Report}
 \setDocRef{\lsstDocType-\lsstDocNum}
 \setDocDate{\vcsdate}
 \setDocUpstreamLocation{\url{https://github.com/lsst/lsst-texmf/examples}}
-\author{% `git log --pretty=%an | sort --key=2 | uniq` ?
-  Michelle Butler
-}
-
-% Most recent last
-\setDocChangeRecord{
-\addtohist{}{2018-10-31}{Test plan Update}{Butler, Comoretto}
-\addtohist{}{2018-10-10}{Initial version.}{Butler, Comoretto}
-}
+\author{ Michelle Butler }
 
-\setDocCurator{Michelle Butler}
-\setDocUpstreamLocation{\url{https://github.com/lsst-dm/\lsstDocType-\lsstDocNum}}
-\setDocUpstreamVersion{\vcsrevision}
+\input history_and_info.tex
 
 
 \setDocAbstract{
-This is the test plan and report for \milestoneId{} (\milestoneName{}), an LSST DM level 1 milestone pertaining to the \product{}.
+This is the test plan and report for \milestoneId{} (\milestoneName{}), an LSST level 2 milestone pertaining to the \product{} Subsystem.
```

Comment:
Archiving isn't a subsystem. See comments on DMTR-91 (https://github.com/lsst-dm/DMTR-91/pull/1) — many of them apply here too.

Reply:
I think that's fine in the long term, but:

- In the short term, the document tree isn't stable.
- They still aren't subsystems! They are products (or services, or deliverables, or whatever) of the Data Management Subsystem.

Reply:
I get the point now. I used the same working as in the manually written DMTRs. I will use the word "deliverable" instead of subsystem.

Reply:
OK, I rephrase it in the template.

Reply:
Done

Reply:
thanks.  I read through DMTR-91.   :-)   I changed wording to reflect that advice.